src/java/org/apache/cassandra/db/compaction/CompactionStrategyManager.java

```diff
@@ -283,6 +284,7 @@ public void resume()
         writeLock.lock();
         try
         {
+            logger.debug("Bereng resume\n" + ExceptionUtils.getStackTrace(new Exception()));
```

Comment:
@bereng remove this please

Reply:
Yep, leftovers to be removed while I was waiting for one of the original authors to validate the approach...